tools/hlint.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+usage() { echo "Usage: $0 -f [all, changeset] -m [check, inplace]" 1>&2; exit 1; }
+
+files=''
+check=true
+
+while getopts ':f:m:' opt
+ do
+     case $opt in
+         f) f=${OPTARG}
+            if [ "$f" = "all" ]; then
+              files=$(find libs/ services/ -not -path "*/test/*" | grep \.hs\$)
```

Comment:
`find` can do the `grep` for you; `-name '*.hs'` should work.

Reply:
Thanks,  I'm not that versed in bash script! :)